README.md

```diff
@@ -97,6 +97,13 @@ OSH_THEME="agnoster" # (this is one of the fancy ones)
 # see https://github.com/ohmybash/oh-my-bash/wiki/Themes#agnoster
 ```
 
+To modify theme automatically, you can use this sed command to edit OSH_THEME.
+
+```shell
+theme="powerline"
+sed -i "s/\(^OSH_THEME=\"\)\([^\"\\n]*\)\(\".*$\)/\1$theme\3/g" ~/.bashrc
```

Comment:
- This usage of `-i` is only supported by GNU sed.
- `g` doesn't have an effect because we are matching using anchor `^` for the beginning of the line.
- This doesn't work when the right-hand side is not enclosed within double quotations.
- I see `\n` in the bracket expression, but newline characters do not appear with this usage because the matching is performed for each line.